### PR TITLE
[EM-2604] Added installment Maybank and deprecated installment Ezypay

### PR DESCRIPTION
**Pull request by Narum11**
* An option for Maybank Installment is selectable on Android SDK.
* Deprecated Installment Ezypay

Screen Shot 2022-11-14 at 2 08 38 PM

### CODE REVIEW

**Review by kan98 — APPROVED** [review]
can you add screenshots of the installment page to the PR as reference?

**Review by new4762 — APPROVED** [review]

**Review by jutinapas — APPROVED** [review]

File: sdk/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -65,7 +65,7 @@
     <string name="payment_method_credit_card_title">クレジットカード</string>
     <string name="payment_method_installment_bay_title">クルンシィ</string>
     <string name="payment_method_installment_bbl_title">バンコク銀行</string>
-    <string name="payment_method_installment_ezypay_title">メイバンクイージーペイ</string>
+    <string name="payment_method_installment_mbb_title">メイバンクイージーペイ</string>
```
Comment (hpariwat) [inline sdk/src/main/res/values-ja/strings.xml:68]: Could you reduce it to be just this one `メイバンク`?

**Review by hpariwat — APPROVED** [review]

**Review by nikolack — APPROVED** [review]

**Comment by Narum11** [comment]
> can you add screenshots of the installment page to the PR as reference?

Added ka :slightly_smiling_face:

**Review by nuxzero — APPROVED** [review]
Seems this PR contains breaking changes. However, please inform merchants when we release a new version.

**Comment by Narum11** [comment]
> Seems this PR contains breaking change

For more information, Installment Maybank is the successor of the unused installment Ezypay ka 🙂

**Pull request merged**